src/lib/components/Swap/Toolbar/Caption.tsx

```diff
@@ -63,21 +66,21 @@ export function WrapCurrency({ loading, wrapType }: { loading: boolean; wrapType
 
 export function Trade({ trade }: { trade: InterfaceTrade<Currency, Currency, TradeType> }) {
   const [flip, setFlip] = useState(true)
-  const { inputAmount, outputAmount, executionPrice } = trade
-  const fiatValueInput = useUSDCPrice(inputAmount.currency)
-  const fiatValueOutput = useUSDCPrice(outputAmount.currency)
+  const { inputAmount: input, outputAmount: output, executionPrice } = trade
+  const { inputUSDC, outputUSDC, priceImpact } = useUSDCPriceImpact(input, output)
+  const isPriceImpactHigh = priceImpact && getPriceImpactWarning(priceImpact) === 'error'
```

Comment:
This should check if `priceImpact && getPriceImpactWarning(priceImpact)`, as it should be displayed whether the warning state is `warning` _or_ `error`.